tox.ini

```diff
@@ -0,0 +1,18 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py35
```

Comment:
Added. Note that I couldn't figure out how to install both 3.5 and 3.4 so I only test on 3.5 locally.